<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskDecryptionDialog</class>
 <widget class="QDialog" name="MaskDecryptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask Encryption Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLineEdit" name="AlphabetLineEdit">
     <property name="placeholderText">
      <string>abc...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="MaskFromIniPushButton">
     <property name="text">
      <string>From .ini</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="AlphabetFromIniPushButton">
     <property name="text">
      <string>From .ini</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="AlphabetLabel">
     <property name="text">
      <string>Alphabet:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="MaskLineEdit">
     <property name="placeholderText">
      <string>000-000-000...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="MaskLabel">
     <property name="text">
      <string>Mask</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="DecryptionPushButton">
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="MaskFromFilePushButton">
     <property name="text">
      <string>From File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AlphabetLineEdit</tabstop>
  <tabstop>MaskLineEdit</tabstop>
  <tabstop>DecryptionPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
